llama_stack/providers/inline/scoring/llm_as_judge/scoring_fn/fn_defs/llm_as_judge_405b_simpleqa.py

```diff
@@ -0,0 +1,91 @@
+    - Predicted answers "100k" and "113k" are INCORRECT.
+    - Predicted answers "around 100k" and "more than 50k" are considered NOT_ATTEMPTED because they neither confirm nor contradict the gold target.
+- The gold target may contain more information than the question. In such cases, the predicted answer only needs to contain the information that is in the question.
+    - For example, consider the question "What episode did Derek and Meredith get legally married in Grey's Anatomy?" with gold target "Season 7, Episode 20: White Wedding". Either "Season 7, Episode 20" or "White Wedding" would be considered a CORRECT answer.
+- Do not punish predicted answers if they omit information that would be clearly inferred from the question.
+    - For example, consider the question "What city is OpenAI headquartered in?" and the gold target "San Francisco, California". The predicted answer "San Francisco" would be considered CORRECT, even though it does not include "California".
+    - Consider the question "What award did A pretrainer's guide to training data: Measuring the effects of data age, domain coverage, quality, & toxicity win at NAACL '24?", the gold target is "Outstanding Paper Award". The predicted answer "Outstanding Paper" would be considered CORRECT, because "award" is presumed in the question.
+    - For the question "What is the height of Jason Wei in meters?", the gold target is "1.73 m". The predicted answer "1.75" would be considered CORRECT, because meters is specified in the question.
+    - For the question "What is the name of Barack Obama's wife?", the gold target is "Michelle Obama". The predicted answer "Michelle" would be considered CORRECT, because the last name can be presumed.
+- Do not punish for typos in people's name if it's clearly the same name.
+    - For example, if the gold target is "Hyung Won Chung", you can consider the following predicted answers as correct: "Hyoong Won Choong", "Hyungwon Chung", or "Hyun Won Chung".
+Here is a new example. Simply reply with either CORRECT, INCORRECT, NOT ATTEMPTED. Don't apologize or correct yourself if there was a mistake; we are just trying to grade the answer.
+```
+Question: {input_query}
+Gold target: {expected_answer}
+Predicted answer: {generated_answer}
+```
+Grade the predicted answer of this new question as one of:
+A: CORRECT
+B: INCORRECT
+C: NOT_ATTEMPTED
+Just return the letters "A", "B", or "C", with no text around it.
+""".strip()
+
+
+llm_as_judge_405b_simpleqa = ScoringFn(
+    identifier="llm-as-judge::llm_as_judge_405b_simpleqa",
+    description="Llm As Judge Scoring Function for SimpleQA Benchmark (https://github.com/openai/simple-evals/blob/main/simpleqa_eval.py)",
+    return_type=NumberType(),
+    provider_id="llm-as-judge",
+    provider_resource_id="llm-as-judge-405b-simpleqa",
+    params=LLMAsJudgeScoringFnParams(
+        judge_model="Llama3.1-405B-Instruct",
+        prompt_template=GRADER_TEMPLATE,
+        judge_score_regexes=[r"(A|B|C)"],
```

Comment:
is the regex supposed to match the entire text, that is does it have an implicit `^` and `$` around it?

Reply:
Hmm, I think it should have "^(A|B|C)$", since judge prompt to output only A/B/C. But leaving it as this "(A|B|C)" aligns with the original implementation: https://github.com/openai/simple-evals/blob/580d359553a88584c11ce4efb97d49d9386e0d9e/simpleqa_eval.py#L127